integ-test/src/test/java/org/opensearch/sql/ppl/DateTimeFunctionIT.java

```diff
@@ -78,6 +155,105 @@ public void testDateAdd() throws IOException {
     verifySome(result.getJSONArray("datarows"), rows("2020-09-17 17:30:00"));
   }
 
+  @Test
+  public void testDateTime() throws IOException {
```

Comment:
What is the importance of each test case? You could break them out into separate tests (🔥) or at least document each one.